test/CMakeLists.txt

```diff
@@ -14,14 +14,11 @@ set(CIRCT_TEST_DEPENDS
   circt-translate
   handshake-runner
   firtool
+  llhd-sim
   mlir-opt
   mlir-cpu-runner
   )
 
-if(NOT WIN32)
```

Comment:
Do the tests now work on Windows or does it just not cause the Windows build to fail?

Reply:
I'm not sure yet :) I want to trigger the test manually and see what happens.